README.md

```diff
@@ -123,8 +123,7 @@ The Terraform VS Code extension bundles the [Terraform Language Server](https://
 
 The extension does require the following to be installed before use:
 
-- VS Code v1.65 or greater
-- Terraform v0.12 or greater
```

Comment:
I'm not sure if _required_ is the best wording here, but I recall people complaining in the past that we didn't communicate this as a requirement for formatting.

Reply:
I see your point. I have added it back with a little rewording